R/errors.R

```diff
@@ -14,10 +14,16 @@ CheckVariableLengths <- function(variable.list, list.name)
 }
 
 #' Error condition with special class to identify user errors
-#' @param x The message to pass to the user
-#' @param ... Additional arguments to pass to errorCondition
-#' @return An errorCondition with the "UserError" class
+#' @param call. Unused parameter, used to absorb usage from previous calls using stop
```

Comment:
Could you add `@param` for `...` and `domain` as well?

Reply:
They are inherited from `base::stop` via the use of `@inheritParams`. So they will get whatever the base documentation uses for `stop`

Reply:
Can see in the `Rd` file diff.

Reply:
OK